### PR TITLE
[Doc] Update *Overview of Connector*

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix https://github.com/apache/pulsar/issues/5015

### CODE REVIEW

File: site2/docs/io-overview.md [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
+
+## Work with connector
+
+You can manage Pulsar connectors (for example, create, update, start, stop, restart, reload, delete and perform other operations on connectors) via the [Connector CLI](reference-connector-admin.md) with [sources](reference-connector-admin.md#sources) and [sinks](reference-connector-admin.md#sinks) subcommands.
+
+## Relationship between connector and Functions
+
+Pulsar connectors are **special** [Pulsar Functions](functions-overview.md) built to interact with external systems.
+
+Sources, sinks, and Functions are components of instances, and they all run on Functions workers. When managing sources, sinks or Functions via **Pulsar admin CLI** (that is, [Connector CLI](reference-connector-admin.md) and [Functions CLI](functions-cli.md)), an instance is started on a worker.
```
[the remark this answers is not in the document]
Comment (tuteng) [reply site2/docs/io-overview.md]: There is no necessary relationship between `connectors` and `functions`. The relationship between `connectors` and `function worker` can be referred to this figure http://pulsar.apache.org/docs/en/functions-worker/#run-functions-worker-separately.

Comment (tuteng) [inline site2/docs/io-overview.md]: `sources, sinks or Functions` does the first letter have to be uppercase or lowercase?

**Comment by Anonymitaet** [comment]
@tuteng I've incorporated all your comments, could you PTAL?

**Comment by Anonymitaet** [comment]
run integration tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Pull request merged**